### ISSUE TITLE
BC 7.99.17/ MC 1.12.2 - The builder crashes when I insert the blueprint

### ISSUE THREAD

**Issue by eliaguarneri** [BuildCraft/BuildCraft#4167]
BuildCraft version: buildcraft-7.99.17
Forge version: forge-1.12.2-14.23.3.2655
Link to crash report or log: 

(https://gist.github.com/eliaguarneri/d9f6ba902ae2729544055262ccc3ede7)

Singleplayer or multiplayer: Singleplayer
Steps to reproduce: The builder crashes when I insert the blueprint

Additional information: 
[crash-2018-05-17_08.37.07-server.txt](https://github.com/BuildCraft/BuildCraft/files/2012197/crash-2018-05-17_08.37.07-server.txt)

**Comment by eliaguarneri** [comment]
Created with architect table.
I attach screenshots.
Is it possible that it does not work only bluprint?
Template works with the builder.
![architect table zone - 2018-05-17_14 08 14](https://user-images.githubusercontent.com/39366992/40179616-b44d6c12-59dc-11e8-802a-57f3f044297f.png)
![zone builder - 2018-05-17_14 08 52](https://user-images.githubusercontent.com/39366992/40179617-b470c608-59dc-11e8-8978-80dfe330ffec.png)

**Comment by eliaguarneri** [comment]
Crasha with any bluprint I insert.
even with one who had never crashed.
The bluprint now recreate them with the electronic library.
Because unfortunately once the crashed bluprint disappears.

**Comment by afdw** [comment]
Do you have anything non-vanilla (including blocks from BC) in the zone?

**Comment by eliaguarneri** [comment]
No that I find myself in the construction area, far away is a pump as you can see from the screenshot.
How can I understand it?

The problem that a first construction has done after stop!
He has not worked anymore!

I used objects already present in the game and not the mods:
The various wood plank for mosaic parquet internally,
Wood plank for walls,
wooden fence,
glass,
sea lantern,
torches,
doors,
rose,
peonies,
oak leaves,
carpet,
library,
furnace,
slab stone,
skull box,
chest,
beed,
nether brick,
nether brick stairs for the roof.

**Comment by eliaguarneri** [comment]
A question still.
Is it possible to move a blueprint to another world?
If yes, how do I do it?

**Comment by afdw** [comment]
Yes, you can. You can use an Electronic Library. Put the snapshot item in it in one world, then in other world select the name from the list and put an empty snapshot item. You can also transfer snapshots between different Minecraft installations by copying files in `snapshots-client` folders.

**Comment by eliaguarneri** [comment]
I try: D
Thank you I had to uninstall everything now I see if it works :(.

to get from Scotland
